repos/fdbt-site/src/pages/viewPassengerTypes.tsx

```diff
@@ -0,0 +1,278 @@
+            <button className="govuk-button" data-module="govuk-button">
+                Add a passenger type
+            </button>
+        </div>
+    );
+};
+
+const IndividualPassengerTypes = ({ passengerTypes }: { passengerTypes: PassengerType[] }): ReactElement => {
+    return (
+        <div className="govuk-heading-m">
+            <h3>Individual</h3>
+
+            <div className="govuk-grid-row">
+                {passengerTypes.map((d) => (
+                    <div key={d.passengerType} className="govuk-grid-column-one-half govuk-!-margin-bottom-5">
+                        <div className="card">
+                            <div className="card__body">
+                                <div className="card__actions">
+                                    <ul className="actions__list">
+                                        <li className="actions__item">
+                                            <a
+                                                className="govuk-link govuk-!-font-size-19 govuk-!-font-weight-regular"
+                                                href="/viewPassengerTypes"
+                                            >
+                                                Edit
+                                            </a>
+                                        </li>
+
+                                        <li className="actions__item">
+                                            <a
+                                                className="govuk-link govuk-!-font-size-19 govuk-!-font-weight-regular actions__delete"
+                                                href="/viewPassengerTypes"
+                                            >
+                                                Delete
+                                            </a>
+                                        </li>
+                                    </ul>
+                                </div>
+
+                                <h4 className="govuk-!-padding-bottom-4">{sentenceCaseString(d.passengerType)}</h4>
+
+                                <p className="govuk-body-s govuk-!-margin-bottom-2">
+                                    <span className="govuk-!-font-weight-bold">Minimum age:</span>{' '}
+                                    {d.ageRangeMin ? d.ageRangeMin : 'N/A'}
+                                </p>
+
+                                <p className="govuk-body-s govuk-!-margin-bottom-2">
+                                    <span className="govuk-!-font-weight-bold">Maximum age:</span>{' '}
+                                    {d.ageRangeMax ? d.ageRangeMax : 'N/A'}
+                                </p>
+
+                                <p className="govuk-body-s govuk-!-margin-bottom-2">
+                                    <span className="govuk-!-font-weight-bold">Proof document:</span>{' '}
+                                    {d.proofDocuments ? d.proofDocuments : 'N/A'}
```

Comment:
I think this needs formatting - something similar to the fareConfirmation page where we do:
```
passenger.proofDocuments.map((proofDoc) => sentenceCaseString(proofDoc)).join(', ')
```